### DISCUSSION TITLE
umi4如何将mfsu应用到打包构建

### DISCUSSION THREAD

**Discussion by UestcCarpediem** [umijs/umi#11484] [Q&A]
umi3S是可以将mfsu用到打包构建的，但是umi4好像没有相关配置，有没有办法将mfsu如何将mfsu应用到打包构建

**Comment by fz6m** [comment]
umi 4 的 mfsu 只能开发时用，如果打包有瓶颈，又对兼容性要求不高，可以切换一下 [转译器](https://umijs.org/docs/api/config#srctranspiler)，到 swc 或者 esbuid 。

另外，如果你能联动基建和构建容器，可以尝试 [物理构建缓存](https://umijs.org/blog/webpack-5-prod-cache) 。
